### ISSUE TITLE
[Feature Request] Port Oil Rig and X-Ray Skater from the Xbox version

### ISSUE THREAD

**Issue by Patrxgt** [PARTYMANX/partymod-thps3#7]
It would be nice, if you could add Oil Rig and X-Ray Skater from the Xbox version.

And yes, I'm aware, that [LevelMod](http://www.chapter-3.net/thps3/v2/showthread.php?tid=3141) brings back this map, but it doesn't integrates it into the game like Xbox version and [Bonus Skaters Mod](https://iron-hawk.wixsite.com/ironhawk/mods) adds X-Ray Skater, but it also lacks proper integration + shop owner is missing exclusive animations for this one.

**Comment by PARTYMANX** [comment]
this has been on my list for a bit (Japanese skaters as well);  problem is that it would require quite a bit of work to figure out how to legally include it all.  My goal is to avoid any potential issues by never including copyrighted content, so i'd need to add some sort of disc importing mechanism

**Comment by Patrxgt** [comment]
Man, this game is older than most of the GitHub users and wasn't available on sale for almost 2 decades. I don't think, that Activision would go and sue you for including a map and models from other versions of the game. If that would've been the case, then LevelMod and Bonus Skaters Mod wouldn't exist. Hell, many other console-to-PC fan ports wouldn't exist, because they package files from the console versions of the game and just let people download it (even Rockstar isn't banning those kind of mods!). So, as far as I'm concerned, you won't be in the big trouble, if you would include those things.

**Comment by PARTYMANX** [comment]
With Activision taking down some CoD mods in the recent past (as far as i know, for much more extreme forms of distributing game data), I'm not super interested in taking on any sort of risk, as small as it may seem.  Things can change on a dime, especially if a remake were to start production.  This project took some time to develop, and I'm not interested in that going away, this being a platform that's pretty prone to quick takedowns.

**Comment by Patrxgt** [comment]
Well, you could always provide needed assets outside of the GitHub.

**Comment by ZT-1** [comment]
I agree with all the reasons Patrxgt mentioned. Everything aside from 1+2 is pretty much considered abandonware. They haven't taken down anything TH related like Iron Hawk's version of integrating these two things and not to mention thugpro, thpm and rethawed.
